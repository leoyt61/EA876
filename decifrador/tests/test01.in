FEB DWZG XXR WVKO YOR MAJISH XEEZP.
