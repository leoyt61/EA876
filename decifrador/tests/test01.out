NAO DEVE TER SIDO UMA TAREFA FACIL.
